pkg/validations/commandvalidations/commandvalidations.go

```diff
@@ -0,0 +1,82 @@
+package commandvalidations
```

Comment:
should this package be `cluster`? or `createcluster`? (one word is always better than two but not sure if `cluster` is too generic

that way we get `pkg/validations/createcluster`